tests/release_scraper/test_release_scraper.py

```diff
@@ -32,6 +32,12 @@
     ({'platform': 'win32', 'version': '23.0.1'},
      'firefox-23.0.1.en-US.win32.exe',
      'firefox/releases/23.0.1/win32/en-US/Firefox Setup 23.0.1.exe'),
+    ({'platform': 'win32', 'version': '23'},
+     'firefox-23.1.0.en-US.win32.exe',
+     'firefox/releases/23.1.0/win32/en-US/Firefox Setup 23.1.0.exe'),
+    ({'platform': 'win32', 'version': '23.0'},
```

Comment:
Hm, for this specific test I would actually expect that the 23.0.1 release would be downloaded which is the latest security release of the 23.0 minor version. And we have this release already present for the testing data.

Reply:
Well, the test doesn't specify `23` but is more strict in requesting a version for `23.0`. That means that we cannot allow a download of `23.1.x` but have to stick to the minor version which is `23.0.x` and as such we still need to return `23.0.1`.